src/date.rs

```diff
@@ -20,15 +20,17 @@ impl Date {
     const PLIST_EPOCH_UNIX_TIMESTAMP: Duration = Duration::from_secs(978_307_200);
 
     pub(crate) fn from_rfc3339(date: &str) -> Result<Self, ()> {
-        let offset: DateTime<FixedOffset> = DateTime::parse_from_rfc3339(date).map_err(|_| ())?;
+        let offset: OffsetDateTime = OffsetDateTime::parse(date, &Rfc3339)
+            .map_err(|_| ())?
+            .to_offset(UtcOffset::UTC);
         Ok(Date {
-            inner: offset.with_timezone(&Utc).into(),
+            inner: offset.into(),
         })
     }
 
     pub(crate) fn to_rfc3339(&self) -> String {
-        let datetime: DateTime<Utc> = self.inner.into();
-        datetime.to_rfc3339_opts(SecondsFormat::Secs, true)
+        let datetime: OffsetDateTime = self.inner.into();
+        return datetime.format(&Rfc3339).unwrap_or_default();
```

Comment:
This should probably just be an unwrap(). I can't see why it would fail, and if it does I'd rather panic than output an invalid plist.